packages/riverpod/test/devtool_test.dart

```diff
@@ -0,0 +1,218 @@
+// ignore_for_file: avoid_types_on_closure_parameters
+
+import 'package:riverpod/src/internals.dart';
+import 'package:test/test.dart';
+
+void main() {
+  tearDown(() {
+    RiverpodBinding.debugInstance.containers = {};
+    stopSpyPostEvent();
+  });
+
+  test('should add container to containers map', () {
+    final firstContainer = ProviderContainer();
```

Comment:
use the `createContainer` utility instead of manually creating ProviderContainers

Reply:
done